### PR TITLE
SRCH-3434 expose JS indexing option to domains table for super admins

### PR DESCRIPTION
**Pull request by krbhavith**
## Summary
- New "Edit" button to update Searchgov Domains for Super Admin. So that admin can access functionality of enable/disable ''js_renderer" from UI for Searchgov Domains.
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

**Review by lorawoodford — CHANGES_REQUESTED** [review]
@krbhavith This looks good!  A very few minor tweaks in the code (feel free to push back if you disagree with any), but marking as "Request changes" mainly so that @amanifarooque has time to weigh in on what would be helpful labeling for the checkbox.  I'll defer to her on that, aside from saying that I don't think the default "Js renderer" label constructed from the boolean is clear:

Screen Shot 2022-11-17 at 10 59 27 AM

File: app/controllers/admin/searchgov_domains_controller.rb
Comment (krbhavith) [reply app/controllers/admin/searchgov_domains_controller.rb:24]: Agreed that the label should be more readable. But when I have checked the Affiliates settings, labels are used mostly from columns from that table.

Comment (amanifarooque) [reply app/controllers/admin/searchgov_domains_controller.rb:24]: We don't need the description, but waiting for confirmation from @dawnpm on the label.

Comment (amanifarooque) [reply app/controllers/admin/searchgov_domains_controller.rb:24]: @krbhavith Got confirmation from Dawn, let's label this "Render Javascript".

**Review by lorawoodford — APPROVED** [review]
LGTM

**Pull request merged**
